fecgov/fec-cms
# Feature/2700 fec monospace currency font

## Summary

- Resolves #2700 
Added the font files (v1) beside our other fonts, created a style (`t-mono`) we can apply to any stacks of numbers, applied that class name to all the stacks of cash I could find.

## Impacted areas of the application
Code-wise, only the font files, styles, and various class names were affected. The class was added to several columns across the site.

## Screenshots
Before on the left, after on the right. Notice how the numbers stack more neatly on the right. A perfect example is the 97, 20, and 18 from the first three amounts.
![image](https://user-images.githubusercontent.com/26720877/58661397-ddb90780-82f5-11e9-8ca8-69c43f516024.png)

## Related PRs
None

## How to test
- Pull the branch
- `npm run build`
- Scour the site for neat stacks of cash, here are a few:
  - http://localhost:8000/data/raising-bythenumbers/
  - http://localhost:8000/data/spending-bythenumbers/
  - http://localhost:8000/data/loans/
  - http://localhost:8000/data/candidate/S0NM00058/
- Verify that all currency columns have the monospace font applied
- Keep an eye out for any cells that are now monospace and should _not_ be
- Does the 'files changed' list look complete?

**NOTE**: If the mono font (`t-mono` class name) is applied to any cells that have non-number, non-punctuation characters, those alpha characters will _not_ display at all.
____
